pkg/network/ebpf/c/runtime/tracer.c

```diff
@@ -255,20 +248,63 @@ int kretprobe__tcp_close(struct pt_regs* ctx) {
 
 SEC("kprobe/ip6_make_skb")
 int kprobe__ip6_make_skb(struct pt_regs* ctx) {
+#ifdef FEATURE_IPV6_ENABLED
```

Comment:
Could we move this to before line 249 to completely compile this out if this flat is not set?

Reply:
Turns out, yes! I've updated the code.